packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableTokenImage.tsx

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from 'react'
+import Image from 'next/image'
+import EthereumLogoRoundLight from '@/images/EthereumLogoRoundLight.svg'
+
+import { useTokenLists } from '../../hooks/useTokenLists'
+import { ChainId } from '../../util/networks'
+import { ether } from '../../constants'
+
+export const TransactionsTableTokenImage = ({
+  tokenSymbol
+}: {
+  tokenSymbol: string
+}) => {
+  // we need to take token image from mainnet by symbol, some token images don't exists on other networks
+  const tokenLists = useTokenLists(ChainId.ArbitrumOne)
+
+  const allTokens = useMemo(() => {
+    return tokenLists.data?.map(t => t.tokens).flat() || []
+  }, [tokenLists])
+
+  const token = useMemo(() => {
+    return allTokens.find(
+      t => t.symbol.toLowerCase() === tokenSymbol.toLowerCase()
+    )
+  }, [allTokens, tokenSymbol])
+
+  if (tokenSymbol.toLowerCase() === ether.symbol.toLowerCase()) {
+    return (
+      <Image
+        height={20}
+        width={20}
+        alt="ETH logo"
+        src={EthereumLogoRoundLight}
+      />
+    )
+  }
+
+  if (!token || !token.logoURI) {
+    return null
+  }
+
+  return (
+    // SafeImage is used for token logo, we don't know at buildtime where those images will be loaded from
+    // It would throw error if it's loaded from external domains
+    // eslint-disable-next-line @next/next/no-img-element
+    <img
```

Comment:
Can't we use `Image from next/image` to achieve the same thing? It takes care of internal optimizations.

Reply:
Getting errors there 
<img width="935" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/d35b5738-d899-4d59-90d2-0dffb17e38df">

Reply:
Okay, so we need to whitelist all these URL's to make external fetches work, then I think the existing approach is fine.